src/gtest/test_rpc_wallet.cpp

```diff
@@ -1,1258 +1,1274 @@
-        "mainnet memo");
-
-    vector<MergeToAddressInputUTXO> utxoInputs;
-    vector<MergeToAddressInputSaplingNote> saplingNoteInputs;
-    try {
-        auto operation = make_shared<AsyncRPCOperation_mergetoaddress>(nullopt, mtx, utxoInputs, saplingNoteInputs, testnetzaddr, -1);
-        BOOST_FAIL("Should have caused an error");
-    } catch (const UniValue& objError) {
-        BOOST_CHECK( find_error(objError, "Fee is out of range"));
-    }
-
-    try {
-        auto operation = make_shared<AsyncRPCOperation_mergetoaddress>(nullopt, mtx, utxoInputs, saplingNoteInputs, testnetzaddr, 1);
-        BOOST_FAIL("Should have caused an error");
-    } catch (const UniValue& objError) {
-        BOOST_CHECK( find_error(objError, "No inputs"));
-    }
-
-    utxoInputs.emplace_back(COutPoint(uint256(), 0), 0, CScript());
-    try {
-        MergeToAddressRecipient badaddr("", "memo");
-        auto operation = make_shared<AsyncRPCOperation_mergetoaddress>(nullopt, mtx, utxoInputs, saplingNoteInputs, badaddr, 1);
-        BOOST_FAIL("Should have caused an error");
-    } catch (const UniValue& objError) {
-        BOOST_CHECK( find_error(objError, "Recipient parameter missing"));
-    }
-
-    // Testnet payment addresses begin with 'tZ'.  This test detects an incorrect prefix.
-    try {
-        auto operation = make_shared<AsyncRPCOperation_mergetoaddress>(nullopt, mtx, utxoInputs, saplingNoteInputs, mainnetzaddr, 1);
-        BOOST_FAIL("Should have caused an error");
-    } catch (const UniValue& objError) {
-        BOOST_CHECK( find_error(objError, "Invalid recipient address"));
-    }
-
-    // Un-set global state
-    fExperimentalMode = false;
-    mapArgs.erase("-zmergetoaddress");
-}
-
-BOOST_AUTO_TEST_SUITE_END()
+// Copyright (c) 2013-2014 The Bitcoin Core developers
+// Copyright (c) 2021 The Pastel developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php.
+
+#include <array>
+#include <chrono>
+#include <optional>
+#include <thread>
+#include <variant>
+#include <fstream>
+#include <unordered_set>
+
+#include <gtest/gtest.h>
+
+#include <rpc/server.h>
+#include <rpc/client.h>
+#include <fs.h>
+#include <key_io.h>
+#include <main.h>
+#include <wallet/wallet.h>
+#include <zcash/Address.hpp>
+#include <asyncrpcqueue.h>
+#include <asyncrpcoperation.h>
+#include <wallet/asyncrpcoperation_mergetoaddress.h>
+#include <wallet/asyncrpcoperation_sendmany.h>
+#include <wallet/asyncrpcoperation_shieldcoinbase.h>
+#include <init.h>
+#include <utiltest.h>
+#include <tinyformat.h>
+#include <univalue.h>
+#include <test_rpc.h>
+#include <pastel_gtest_main.h>
+
+using namespace std;
+using namespace testing;
+
+extern UniValue createArgs(int nRequired, const char* address1 = nullptr, const char* address2 = nullptr);
+extern UniValue CallRPC(string args);
+
+extern CWallet* pwalletMain;
```

Comment:
this extern is declared already in init.h, we don't need it here... don't know why it was there in rpc_wallet_tests.cpp